config/norns-jack.service

```diff
@@ -8,8 +8,7 @@ Group=we
 Environment="JACK_NO_AUDIO_RESERVATION=1"
 LimitRTPRIO=95
 LimitMEMLOCK=infinity
-ExecStart=/usr/bin/jackd -R -P 95 -d alsa -d hw:0 -r 48000 -n 3 -p 128 -S -s
-ExecStartPost=/bin/sleep 2
+ExecStart=/usr/bin/jackd -R -P 95 -d alsa -d hw:0 -r 48000 -n 3 -p 128 -S
```

Comment:
I'm reluctant to remove the `-s` from this line since we had issues with xruns coming from the driver. If we have some changes/fixes for that problem somewhere that alleviate this problem we might be able to remove this but I don't think we have any fixes like that at the moment (your research into performance/interrupts might be one of those changes for example).

Reply:
fair point! better to leave it even if *I* haven't seen the alsa driver bug....